# HG changeset patch
# User Rik <rik@octave.org>
# Date 1449941639 28800
#      Sat Dec 12 09:33:59 2015 -0800
# Node ID baa3b9f0e9d408a07f92e28d7f123be6993b7dff
# Parent  2bd3b13e2c8ec503d038005c06f1a7513af240b7
2015 Code Sprint: __glpk__.cc: use ovl().

diff --git a/libinterp/dldfcn/__glpk__.cc b/libinterp/dldfcn/__glpk__.cc
--- a/libinterp/dldfcn/__glpk__.cc
+++ b/libinterp/dldfcn/__glpk__.cc
@@ -624,20 +624,17 @@ Undocumented internal function.\n\
     {
       extra.assign ("lambda", lambda);
       extra.assign ("redcosts", redcosts);
     }
 
   extra.assign ("time", time);
   extra.assign ("status", status);
 
-  retval(3) = extra;
-  retval(2) = errnum;
-  retval(1) = fmin;
-  retval(0) = xmin;
+  retval = ovl (xmin, fmin, errnum, extra);
 
 #else
 
   gripe_not_supported ("glpk");
 
 #endif
 
   return retval;
